homeassistant/components/lightwave/__init__.py

```diff
@@ -73,13 +69,15 @@
             async_load_platform(hass, "switch", DOMAIN, switches, config)
         )
 
-    trvs = config[DOMAIN]["trv"]
-    if trvs:
+    trv = config[DOMAIN][CONF_TRV]
+    if trv:
+        hass.data[LIGHTWAVE_TRV_PROXY] = trv[PROXY_IP]
+        hass.data[LIGHTWAVE_TRV_PROXY_PORT] = trv[PROXY_PORT]
         hass.async_create_task(
-            async_load_platform(hass, "climate", DOMAIN, trvs, config)
+            async_load_platform(hass, "climate", DOMAIN, trv[CONF_TRVS], config)
```

Comment:
Please create a variable beforehand for `trv[CONF_TRVS]` and use it here and in line 80.